### PR TITLE
Adjust PSQL sidecar limit

### PR DESCRIPTION
**Pull request by wejdross**
## Checklist

- [x] The PR has a meaningful title. It will be used to auto generate the
      changelog.
      The PR has a meaningful description that sums up the change. It will be
      linked in the changelog.
- [x] PR contains a single logical change (to build a better changelog).
- [x] Categorize the PR by adding one of the labels:
      `bug`, `enhancement`, `documentation`, `change`, `breaking`, `dependency`
      as they show up in the changelog.
- [x] Link this PR to related issues or PRs.

### CODE REVIEW

File: component/class/defaults.yml [hunk 1]
```diff
@@ -42,7 +42,7 @@ parameters:
       appcat:
         registry: ghcr.io
         repository: vshn/appcat
-        tag: v4.49.0
+        tag: upgrade/stackgres_version
```
Comment (TheBigLee) [inline component/class/defaults.yml:45]: This needs to be fixed before merging

**Review by TheBigLee — APPROVED** [review]
Please see comment, otherwise LGTM

File: component/class/defaults.yml [hunk 2]
```diff
@@ -457,7 +457,7 @@ parameters:
                 memory: "30Mi"
               limits:
                 cpu: "32m"
-                memory: "2Gi"
+                memory: "64Mi"
```
Comment (TheBigLee) [inline component/class/defaults.yml:460]: This actually might be too low. Have you verified that it works with this?

Comment (Kidswiss) [reply component/class/defaults.yml:460]: I agree, I would go with 256 or something.

**Pull request closed without merging**